dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -28,6 +28,9 @@ connectionManager.desc=Connection Manager configuration
 agedTimeout=Aged timeout
 agedTimeout.desc=Amount of time before a connection can be discarded by pool maintenance. A value of -1 disables this timeout. A value of 0 discards every connection, which disables connection pooling.
 
+abortLongRunningInuseConnections=Abort long running in use connections
```

Comment:
Replace "abort" with "cancel," "stop," "end," or "fail."

Reply:
If we externalize/document this capability, I would like to call it maxInUseTime "Maximum In Use Time" (or pick some other synonym for "in use", in order to align with our other connection manager properties like maxIdleTime and maxPoolSize.